src/prefect/environments/storage/local.py

```diff
@@ -0,0 +1,97 @@
+import cloudpickle
+import os
+from slugify import slugify
+from typing import Any, Dict, Iterable, List, TYPE_CHECKING, Union
+
+import prefect
+from prefect.environments.storage import Storage
+
+if TYPE_CHECKING:
+    from prefect.core.flow import Flow
+
+
+class LocalStorage(Storage):
+    """
+    Local Storage class.  This class represents the Storage
+    interface for Flows stored as bytes in the local filesystem.
+
+    Args:
+        - directory (str, optional): the directory the flows will be stored in;
+            defaults to `~/.prefect/flows`.  If it doesn't already exist, it will be
+            created for you.
+    """
+
+    def __init__(self, directory: str = "~/.prefect/flows") -> None:
+        self.flows = dict()  # type: Dict[str, str]
+        abs_directory = os.path.abspath(os.path.expanduser(directory))
+        if not os.path.exists(abs_directory):
+            os.makedirs(abs_directory)
```

Comment:
Doesn't matter, but this also accepts `exists_ok=True`

Reply:
Ah yes, forgot about that - happy to change if you want!

Reply:
Nah, this is totally fine, was mentioning in case you didn't know